designer/client/src/components/comment/ProcessComments.tsx

```diff
@@ -73,16 +73,16 @@ function ProcessComments(): JSX.Element {
                         {!isLastComment(index) && <ListSeparator />}
                     </div>
                 ))}
-            </ul>
+            </ProcessCommentsList>
             {capabilities.write ? (
-                <div className="add-comment-panel">
+                <AddCommentPanel>
                     <CommentInput onChange={onInputChange.bind(this)} value={comment} />
-                    <NkButton type="button" className="add-comment" onClick={_addComment} disabled={pending || comment == ""}>
+                    <CommentNKButton type="button" onClick={_addComment} disabled={pending || comment == ""}>
```

Comment:
Do we need this NK in the name? Maybe `CommentButton` will be enough?

Reply:
done